script/validate_json.rb

```diff
@@ -70,11 +78,31 @@ def validate_website_entry(key, i)
     validate_localized_urls(key)
 end
 
+def validate_language_key(keys_in_language_json, key, file)
```

Comment:
Given that this function mutates `keys_in_language_json`, I would prefer it be named in a way which makes it clear it's not a pure function. `add_valid_language_key` might be a better name?

Reply:
Agreed!